certstrap/tasks/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -o errexit -o nounset -o xtrace
+
+export GOPATH=$PWD/go
+
+package="github.com/square/certstrap"
+go build -ldflags='-s -w' "${package}"
```

Comment:
Actually, `-s` didn't actually save any bytes at all, so took that out in 23e9e8f90105f2d3d0dd0c8ad0229acaa68157ab. `-w` is still there.